### ISSUE TITLE
[Bug] Partial model initialization when fine-tuning tts_models--en--ljspeech--vits

### ISSUE THREAD

**Issue by yang1fan2** [coqui-ai/TTS#2553]
### Describe the bug

I was trying to fine-tune the model fine-tuning tts_models--en--ljspeech--vits. But got Partial model initialization.

Am I using the wrong version of TTS? Thanks

### To Reproduce

CUDA_VISIBLE_DEVICES="0" python train_vits.py \
    --restore_path  /home/ubuntu/.local/share/tts/tts_models--en--ljspeech--vits/model_file.pth

### Expected behavior

Load the model checkpoint of all parameters.

### Logs

```shell
> Restoring from model_file.pth ...                                                                                                             
 > Restoring Model...                                                                                                                            
 > Partial model initialization...                                                                                                               
 | > Layer missing in the checkpoint: disc.nets.0.convs.0.bias                                                                                   
 | > Layer missing in the checkpoint: disc.nets.0.convs.0.weight_g                                                                               
 | > Layer missing in the checkpoint: disc.nets.0.convs.0.weight_v                                                                               
 | > Layer missing in the checkpoint: disc.nets.0.convs.1.bias                                                                                   
 | > Layer missing in the checkpoint: disc.nets.0.convs.1.weight_g                                                                               
 | > Layer missing in the checkpoint: disc.nets.0.convs.1.weight_v                                                                               
 | > Layer missing in the checkpoint: disc.nets.0.convs.2.bias                                                                                   
 | > Layer missing in the checkpoint: disc.nets.0.convs.2.weight_g                                                                               
 | > Layer missing in the checkpoint: disc.nets.0.convs.2.weight_v                                                                               
 | > Layer missing in the checkpoint: disc.nets.0.convs.3.bias                                                                                   
 | > Layer missing in the checkpoint: disc.nets.0.convs.3.weight_g                                                                               
 | > Layer missing in the checkpoint: disc.nets.0.convs.3.weight_v
 | > Layer missing in the checkpoint: disc.nets.0.convs.4.bias
 | > Layer missing in the checkpoint: disc.nets.0.convs.4.weight_g
 | > Layer missing in the checkpoint: disc.nets.0.convs.4.weight_v
 | > Layer missing in the checkpoint: disc.nets.0.convs.5.bias
 | > Layer missing in the checkpoint: disc.nets.0.convs.5.weight_g
 | > Layer missing in the checkpoint: disc.nets.0.convs.5.weight_v
 | > Layer missing in the checkpoint: disc.nets.0.conv_post.bias
 | > Layer missing in the checkpoint: disc.nets.0.conv_post.weight_g
 | > Layer missing in the checkpoint: disc.nets.0.conv_post.weight_v
 | > Layer missing in the checkpoint: disc.nets.1.convs.0.bias
 | > Layer missing in the checkpoint: disc.nets.1.convs.0.weight_g
 | > Layer missing in the checkpoint: disc.nets.1.convs.0.weight_v
 | > Layer missing in the checkpoint: disc.nets.1.convs.1.bias
 | > Layer missing in the checkpoint: disc.nets.1.convs.1.weight_g
...
 | > Layer missing in the checkpoint: disc.nets.5.convs.0.weight_v                                                                               
 | > Layer missing in the checkpoint: disc.nets.5.convs.1.bias                                                                                   
 | > Layer missing in the checkpoint: disc.nets.5.convs.1.weight_g                                                                               
 | > Layer missing in the checkpoint: disc.nets.5.convs.1.weight_v                                                                               
 | > Layer missing in the checkpoint: disc.nets.5.convs.2.bias                                                                                   
 | > Layer missing in the checkpoint: disc.nets.5.convs.2.weight_g                                                                               
 | > Layer missing in the checkpoint: disc.nets.5.convs.2.weight_v                                                                               
 | > Layer missing in the checkpoint: disc.nets.5.convs.3.bias                                                                                   
 | > Layer missing in the checkpoint: disc.nets.5.convs.3.weight_g                                                                               
 | > Layer missing in the checkpoint: disc.nets.5.convs.3.weight_v                                                                               
 | > Layer missing in the checkpoint: disc.nets.5.convs.4.bias                                                                                   
 | > Layer missing in the checkpoint: disc.nets.5.convs.4.weight_g                                                                               
 | > Layer missing in the checkpoint: disc.nets.5.convs.4.weight_v                                                                               
 | > Layer missing in the checkpoint: disc.nets.5.conv_post.bias                                                                                 
 | > Layer missing in the checkpoint: disc.nets.5.conv_post.weight_g
 | > Layer missing in the checkpoint: disc.nets.5.conv_post.weight_v
 | > Layer dimention missmatch between model definition and checkpoint: text_encoder.emb.weight
 | > 837 / 949 layers are restored.
 > Model restored from step 1000000

 > Model has 83059180 parameters
```
```


### Environment

```shell
{
    "CUDA": {
        "GPU": [
            "NVIDIA A10G"
        ],
        "available": true,
        "version": "11.7"
    },
    "Packages": {
        "PyTorch_debug": false,
        "PyTorch_version": "2.0.0+cu117",
        "TTS": "0.13.2",
        "numpy": "1.23.5"
    },
    "System": {
        "OS": "Linux",
        "architecture": [
            "64bit",
            "ELF"
        ],
        "processor": "x86_64",
        "python": "3.10.9",
        "version": "#37~20.04.1-Ubuntu SMP Fri Mar 17 11:39:30 UTC 2023"
    }
}
```

**Comment by lexkoro** [comment]
The model `tts_models--en--ljspeech--vits` has no discriminator layers in the checkpoint.

**Comment by yang1fan2** [comment]
@lexkoro Thanks for the response. I downloaded the model checkpoint directly from coqui/tts. Is there any checkpoint to download, which contains the discriminator layer?

**Comment by erogol** [comment]
that model has no disc sorry.

**Comment by codepharmer** [comment]
Is that what is causing these issues? I'm facing similar problems on Tacotron2, but I don't think it uses a GAN.

**Comment by codepharmer** [comment]
@yang1fan2 
Were you ever aboutt o figure this out?

**Comment by lexkoro** [comment]
@codepharmer As mentioned above the saved checkpoint doesn't have the discriminator layers in it. 
If you are facing it for Tacotron2 it could mean the same, the checkpoint you are using doesn't have the layers for the model you are trying to load it for. Depending on the use-case it doesn't have to be bad.

**Comment by abdouaziz** [comment]
> @yang1fan2 Were you ever able to figure this out?

Recently i face the same problem , i solved by downgrading the tts version to 0.19
